tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -12012,9 +12013,9 @@ void ExecutionTest::runCoopVecMulTest() {
 #else
   // Create device and verify coopvec support
   CComPtr<ID3D12Device> D3DDevice;
-  if (!CreateDevice(&D3DDevice, D3D_SHADER_MODEL_6_9)) {
+  if (!CreateDevice(&D3DDevice, D3D_SHADER_MODEL_6_9))
     return;
```

Comment:
I know you're copying the pattern seen elsewhere in this code. But this will result in a skip when this test is integrated into the HLK.

There's no way you would be aware of this, but skip logic isn't properly supported in the HLK. The test will effectively become a pass.


You can fix this using the _HLK_CONF ifdef:

  if (!CreateDevice(&D3DDevice, D3D_SHADER_MODEL_6_9, false)) {
#ifdef _HLK_CONF
    LOG_ERROR_FMT_THROW(L"Device does not support SM 6.9. Can't run these tests.");
#else
    WEX::Logging::Log::Comment(
        "Device does not support SM 6.9. Can't run these tests.");
    WEX::Logging::Log::Result(WEX::Logging::TestResults::Skipped);
    return;
#endif
  }